### DISCUSSION TITLE
UI sucks, honest first time user

### DISCUSSION THREAD

**Discussion by CruiseMan** [cryptoadvance/specter-desktop#1268] [Help]
Let me put it simple, written to disk `0` but written in Activity Monitor ` 21GB`, why ?
<img width="876" alt="Screen Shot 2021-06-21 at 10 47 55 PM" src="https://user-images.githubusercontent.com/15807643/122870920-24199e80-d2e3-11eb-9066-79b7c26fd1fc.png">

**Comment by bitpaint** [comment]
Let me put it simple: wait for the full blockchain to be downloaded before opening Issues.
Why did you not include this part on the screenshot?
![image](https://user-images.githubusercontent.com/67663265/124117076-f9acab80-da6f-11eb-806c-cdad55f0b105.png)

**Comment by k9ert** [comment]
I'm not sure what you exactly expect here but as @bitpaint said, doesn't look very clearly like a bug. Shifting to discussions ...
